layouts/index.html

```diff
@@ -1,12 +1,17 @@
 {{ define "main" }}
 <div>
 
-  <h4>Linux:</h4>
-  <pre><code>curl -L https://github.com/franalgaba/cairo-installer/raw/main/bin/cairo-installer | bash</code></pre>
-
-  <h4>MacOS:</h4>
+  <h4>MacOS</h4>
   Download custom Homebrew formulae and use it to install cairo-lang:
-<pre><code>wget -O /tmp/cairo-lang.rb https://raw.githubusercontent.com/lambdaclass/cairo-by-example/main/homebrew/cairo-lang.rb && brew install --formula --build-from-source /tmp/cairo-lang.rb</code></pre>
+  <pre><code>curl -o /tmp/cairo-lang.rb https://raw.githubusercontent.com/lambdaclass/cairo-by-example/main/homebrew/cairo-lang.rb && brew install --formula --build-from-source /tmp/cairo-lang.rb</code></pre>
+
+  <h4>Arch Linux</h4>
+  Download the package with your favourite AUR helper, such as:
```

Comment:
Maybe add a link to yay, just as an example, for instance:
> Download the package with your fauvorite AUR helper, such as [yay](https://github.com/Jguer/yay):

Reply:
Fixed!